azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -380,19 +375,7 @@ def _run_op(self, op):
 
             @pipeline_thread.runs_on_pipeline_thread
             def on_operation_complete(op, error):
-                if error:
```

Comment:
is removing all error logging from this part of the code the best?

Reply:
you're right.  I'm going to put this back in.

---
In reply to: [369770005](https://github.com/Azure/azure-iot-sdk-python/pull/443#discussion_r369770005) [](ancestors = 369770005)